### PR TITLE
Refactor formatter tests

### PR DESCRIPTION
**Pull request by ybiquitous**
I think that using `stripIndent()` or adding extra indent for the expected output of the formatters leads to inaccuracy and poor readability.
So, this change removes needless indent from the expected output in `lib/formatters/__tests__/*.test.js`.

> Which issue, if any, is this issue related to?

I noticed the problem when I was looking into the source code for #4818.
(though the issue is not directly related to this PR)

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: lib/formatters/__tests__/verboseFormatter.test.js [hunk 1]
```diff
@@ -18,12 +17,13 @@ describe('verboseFormatter', () => {
...
-      1 source checked
-       path/to/file.css
+		expect(output).toBe(
+			`
+1 source checked
+ path/to/file.css
 
-      0 problems found
-    `);
+0 problems found`.trimStart(),
```
Comment (ybiquitous) [inline lib/formatters/__tests__/verboseFormatter.test.js:25]: [note] Reluctantly, I use `trimStart()` only in this file in order to trim the newline at the beginning of the string.

**Review by hudochenkov — COMMENTED** [review]
Indeed it look closer to reality now!

I think it would be great to use `.trimStart()` in every result with more than one line. This way we could see that all lines aligned.

**Comment by ybiquitous** [comment]
@hudochenkov Make sense! I've used `.trimStart()` for the importance of readability, as possible (414f3d7).

**Review by hudochenkov — APPROVED** [review]

**Review by jeddy3 — APPROVED** [review]
Thanks for making these code cleanliness improvements in this pull request and others!

**Pull request merged**